src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react'
 import { Grid, Navbar } from 'react-bootstrap'
-import { InfoPanel, NNBMap, POIForm } from './components'
+import { InfoPanel, NNBMap, POIForm, Counter } from './components'
```

Comment:
Remove all the counter stuff (it actually results in a compile error because the `Counter` component isn't defined)